### DISCUSSION TITLE
urls are not working in selenium grid chrome

### DISCUSSION THREAD

**Discussion by praveen222** [webdriverio/webdriverio#7072] [Q&A]
![Seleniumgrid-Browser](https://user-images.githubusercontent.com/46008743/123740090-6fc0df00-d8c5-11eb-938f-5afee6e73091.png)
Hi,

Urls are not loading in selenium grid Chrome.
any idea how to provide certs or by -pass chrome certs?

screenshot attached

**Comment by christian-bromann** [comment]
@praveen222 the screenshot doesn't help much, can you elaborate on your problem and maybe provide a reproducible example?
